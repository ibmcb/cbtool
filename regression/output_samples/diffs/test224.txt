 status: Waiting for vm_5 (cloud-assigned uuid EBFA4F47-9B41-5129-B2E9-25FDCC7550F0) to start...
 status: Trying to establish network connectivity to vm_5 (cloud-assigned uuid EBFA4F47-9B41-5129-B2E9-25FDCC7550F0), on IP address 15.46.154.151...
 status: Bypassing the sending of a copy of the code tree to vm_5 (15.46.154.151)...
 status: Bypassing generic VM post_boot configuration on vm_5 (15.46.154.151)...
VM object 4342908A-B0DF-5AE5-A5FA-75EEA7F45F4A (named "vm_5") sucessfully attached to this experiment. It is ssh-accessible at the IP address 15.46.154.151 (EBFA4F47-9B41-5129-B2E9-25FDCC7550F0.simcloud.com).
|vm_1          |tinyvm              |pico32      |254.56.139.236  |simhosta0                  |SUT                 |none    |none       |64DD0F5F-27CF-50C8-AE2A-0930A713B586
|vm_2          |tinyvm              |pico32      |5.12.152.246    |simhostd4                  |LG                  |none    |none       |15706C7D-A99C-5D18-971F-C567342734B7
|vm_3          |tinyvm              |pico32      |146.175.146.104 |simhostb1                  |SUT                 |none    |none       |D58FFD02-DE0C-5B99-8BC3-E0417C09EC0D
|vm_4          |db2                 |copper32    |208.171.174.214 |simhosta3                  |SUT                 |none    |none       |B40767D7-C607-51C5-8D73-80D241B38A59
|vm_5          |netclient           |platinum64  |15.46.154.151   |simhostb0                  |SUT                 |none    |none       |4342908A-B0DF-5AE5-A5FA-75EEA7F45F4A
|host_name                               |simhostb0                            
|vmc_name                                |simzone_b                            
|Uptime (in seconds)                                  |13565014                      
|Total Connections Received                           |6494861                       
|Total Commands Processed                             |93769862                      
|Number of Keys                                       |328                           
|Storage Size                                         |16112275440.0                 
|Data Size                                            |13822957604.0                 
|Index Size                                           |550318384                     
|Average Object Size                                  |6602.90838586                 
|Collections                                          |60                            
|59038ADB-0C9D-5DC8-8D0F-858C40A339D8 (simzone_b) VM RESERVATIONS|2                             
|64F2E709-607C-5D1E-9ED5-402265D29FB4 (simzone_d) VM RESERVATIONS|1                             
|538D06C6-8C24-51C6-AC34-D9ECDD4133D6 (simzone_c) VM RESERVATIONS|0                             
|CA6252C4-6803-5C5C-BAA8-5BC509EE3CE0 (simzone_a) VM RESERVATIONS|2                             
