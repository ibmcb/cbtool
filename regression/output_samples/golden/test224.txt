 status: Waiting for vm_5 (cloud-assigned uuid A97B55DC-1400-5A7E-8F2C-140A210F16E3) to start...
 status: Trying to establish network connectivity to vm_5 (cloud-assigned uuid A97B55DC-1400-5A7E-8F2C-140A210F16E3), on IP address 150.186.159.48...
 status: Bypassing the sending of a copy of the code tree to vm_5 (150.186.159.48)...
 status: Bypassing generic VM post_boot configuration on vm_5 (150.186.159.48)...
VM object 21907786-3B36-51AE-BBE4-CC8226715FB1 (named "vm_5") sucessfully attached to this experiment. It is ssh-accessible at the IP address 150.186.159.48 (A97B55DC-1400-5A7E-8F2C-140A210F16E3.simcloud.com).
The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |122.45.84.64    |simhosta2                  |SUT                 |none    |none       |E68D0F85-F4DE-52F9-909E-D7A869F3B6C6
|vm_2          |tinyvm              |pico32      |221.77.166.220  |simhostd4                  |LG                  |none    |none       |DD69A92C-E5BE-5D97-B243-232A08CB1147
|vm_3          |tinyvm              |pico32      |241.236.113.152 |simhostb1                  |SUT                 |none    |none       |C4DD0678-86B9-58ED-B134-B489F2B9A132
|vm_4          |db2                 |copper32    |68.18.106.20    |simhosta4                  |SUT                 |none    |none       |699311B9-F25C-5F16-9866-57C4F5257915
|vm_5          |netclient           |platinum64  |150.186.159.48  |simhosta2                  |SUT                 |none    |none       |21907786-3B36-51AE-BBE4-CC8226715FB1

The VM object vm_5, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VM object key)               |value                                
|host_name                               |simhosta2                            
|meta_tags                               |{'a': 'b', 'x': 'y', 'r': '2'}       
|size                                    |platinum64                           
|vmc_name                                |simzone_a                            
|vmc_pool                                |SUT                                  

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |38.96M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |11740256                      
|Total Connections Received                           |6492681                       
|Total Commands Processed                             |92704739                      
|Number of Keys                                       |327                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |13966159872.0                 
|Data Size                                            |12133511672.0                 
|Index Size                                           |502881232                     
|Average Object Size                                  |6346.37721614                 
|Collections                                          |62                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |5                             
|SVMs                                                 |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |5                             
|VMs ARRIVED                                          |5                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |93                            

 ---------------- VMC-WIDE COUNTERS ----------------
|CB27A579-597C-52AC-B148-644E81A7D93C (simzone_d) VM RESERVATIONS|1                             
|D7D3EB8C-8813-524D-9C2D-7FD7626D4BCC (simzone_b) VM RESERVATIONS|1                             
|97FF48D4-05C4-54D7-B1D8-828EED2A2B20 (simzone_a) VM RESERVATIONS|3                             
|A708E2EA-221A-57ED-BAE8-15077C3E391A (simzone_c) VM RESERVATIONS|0                             

